bolparser.py

```diff
@@ -0,0 +1,118 @@
+__author__ = 'Amogh Jalihal'
+
+import json
+import re
+import sys
+
+class bolParser:
+    def __init__(self, bol):
+        self.bol = bol
+        self.Avartan = list()
```

Comment:
Either up here, or by the getter functions, can you add a comment each defining `Avartan`, `Vibhaag` and `Matra`?

Reply:
Updated this in the most recent commit.